__tests__/nc_news.test.js

```diff
@@ -123,3 +123,218 @@ describe('GET /api/articles/:article_id', () => {
+               expect(body.updated_article).toMatchObject(templateArticle);
+            })
+	});
+    it('PATCH:404 should return error if article ID does not exist', () => {
+        return request(app)
+			.patch('/api/articles/9999')
+            .send({ inc_votes: 2 })
+			.expect(404)
+			.then((response) => {
+				expect(response.body.msg).toBe('Entry not found');
+			});
+    });
+    it('PATCH:400 should return error if given invalid article ID', () => {
+        return request(app)
+			.patch('/api/articles/hamsandwich')
+            .send({ inc_votes: 2 })
+			.expect(400)
+			.then((response) => {
+				expect(response.body.msg).toBe('Bad request');
+			});
+    });
+    it('PATCH:400 should return error if given improperly formatted req', () => {
+        return request(app)
+			.patch('/api/articles/1')
+            .send({ inc_votes: "ello" })
+			.expect(400)
+			.then((response) => {
+				expect(response.body.msg).toBe('Bad request');
+			});
+    });
+    it('PATCH:201 should ignore unecessary additional properties', () => {
+        return request(app)
+        .patch('/api/articles/1')
+        .send({
+            inc_votes: 3, 
+            extra: 500,
+            extra2: "Hello fellow youngsters"
+        })
+        .then(({body}) => {
+            const templateArticle =    {
+                  article_id: 1,
+                  title: 'Living in the shadow of a great man',
+                  topic: 'mitch',
+                  author: 'butter_bridge',
+                  body: 'I find this existence challenging',
+                  created_at: '2020-07-09T20:11:00.000Z',
+                  votes: 103,
+                  article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+                }
+            const article = body.updated_article;
+            expect(article).toMatchObject(templateArticle);
+            
+
+        })
+    });
+});
+
+describe('DELETE /api/comments/:comment_id', () => {
+    it('DELETE:204 should return 204 status code', () => {
+		return request(app).delete('/api/comments/1').expect(204);
+	});
+    it('DELETE:204 should delete the given comment from the database', () => {
+        return request(app).delete('/api/comments/10')
+        .then(() => {
+            return request(app).get('/api/articles')
+        })
+        .then(({body}) => {
+            const articleAfterDeletion = {
+                article_id: 3,
+                title: 'Eight pug gifs that remind me of mitch',
+                topic: 'mitch',
+                author: 'icellusedkars',
+                created_at: '2020-11-03T09:12:00.000Z',
+                article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+                comment_count: 1,
+                votes: '0'
+              };
+            expect(body[0]).toMatchObject(articleAfterDeletion);
+        })
+    });
```

Comment:
This test is a little confusing to read. 

We're making a DELETE request to '/api/comments/10' and then a request to GET all '/api/articles'. We don't necessarily need to use a GET request to demonstrate this here, but if you wanted to use a GET request to demonstrate that a comment is no longer linked to a specific article how could we demonstrate this more clearly?

Reply:
Used comment_count instead of whole object